dotnet/src/dotnetframework/GxClasses/Middleware/GXHttpServices.cs

```diff
@@ -473,12 +473,28 @@ public GXOAuthLogout()
 
 		public override void webExecute()
 		{
+			string genexus_agent = localHttpContext.Request.Headers["Genexus-Agent"];
 			try
 			{
-				GxSecurityProvider.Provider.oauthlogout(context);
+				GxSecurityProvider.Provider.oauthlogout(context, out string URL, out short statusCode);
+
+				if (statusCode == 303)
```

Comment:
Instead of statusCode==303,  statusCode== (int)HttpStatusCode.SeeOther coud be used.